nengo_viz/components/netgraph.py

```diff
@@ -42,12 +153,16 @@ def get_parents(self, uid):
         return parents
 
     def update_client(self, client):
+        self.check_for_reload()
```

Comment:
just flagging this as possibly a good thing to have in the config menu, some users might prefer also just having a button saying 'update' or whatever...by 'just flagging this' i'm not sure what i mean. if there's a way to flag it that'd be sweet.